backend/tests/models/utils.py

```diff
@@ -0,0 +1,6 @@
+import sqlalchemy as sa
+from bamboo.database import db
+
+
+def query_count(model):
+    return db.session.scalar(db.select(sa.func.count()).select_from(db.select(model).subquery()))
```

Comment:
This seem to be translated to 
```
SELECT COUNT(*) FROM (SELECT * FROM "model")
```
Isn't there a more straightforward way?

Reply:
Right, the subquery is unnecessary, just removed.